Sources/NIO/BSDSocketAPI.swift

```diff
@@ -296,6 +300,10 @@ extension NIOBSDSocket.Option {
     /// restricted to IPv6 only.
     public static let ipv6_v6only: NIOBSDSocket.Option =
             NIOBSDSocket.Option(rawValue: IPV6_V6ONLY)
+    
+    /// Request that we are passed traffic class details when receiving datagrams.
```

Comment:
Mild nit here: can you note that you didn't make this public because we don't want people using the socket option channel option to set this, but instead the ECN option? Same above.